sources/packages/backend/apps/workers/src/controllers/assessment/_tests_/e2e/assessment.controller-updateNOAStatus.e2e-spec.ts

```diff
@@ -54,7 +54,7 @@ describe("AssessmentController(e2e)-updateNOAStatus", () => {
     expect(expectedAssessment.noaApprovalStatus).toBe(
       AssessmentStatus.completed,
     );
-    const auditUser = await systemUsersService.systemUser();
+    const auditUser = systemUsersService.systemUser;
```

Comment:
I understand the "renaming" here but I would not enforce that every time for upcoming PRs.
IMO using it in the below way would be also acceptable unless the code would be less readable.
For this PR I do belive that it is all looking good 😉 
```ts
expect(expectedAssessment.modifier).toEqual(systemUsersService.systemUser);
```